diff -Nur -x '*.orig' -x '*~' google-perftools-0.98/src/base/linuxthreads.c google-perftools-0.98.new/src/base/linuxthreads.c
--- google-perftools-0.98/src/base/linuxthreads.c	2008-01-09 17:19:28.000000000 +0000
+++ google-perftools-0.98.new/src/base/linuxthreads.c	2009-03-31 13:59:13.340001573 +0000
@@ -49,7 +49,7 @@
 #include <asm/fcntl.h>
 #include <asm/posix_types.h>
 #include <asm/types.h>
-#include <linux/dirent.h>
+#include <dirent.h>
 
 #include "base/linux_syscall_support.h"
 #include "base/thread_lister.h"
